### PR TITLE
chore: fix minor spelling issues and typos in comments and log messages

### PR DESCRIPTION
**Pull request by KonH**
I polish comments for better understanding.
I have doubt about 'mutationtest' term, but in all repos scope 'mutation test' is more related.
I hope, that PR can made project a little better.

### CODE REVIEW

**Comment by KonH** [comment]
Sorry about conflics, looks like something went wrong with my fork.

File: src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs [hunk 1]
```diff
@@ -67,7 +67,7 @@ public SyntaxNode Mutate(SyntaxNode currentNode)
                 StatementSyntax ast = statement as StatementSyntax;
 
                 // this is a temporary fix for mutating LocalDeclarationStatements because mutating withing these statements breaks the application
-                // TODO: backlog item Ternary assignment statements mutaties
+                // TODO: backlog item Ternary assignment statements mutates
```
Comment (Mobrockers) [inline src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs]: the word you corrected means mutations in dutch so I'm pretty sure this should be mutations and not mutates

**Comment by Mobrockers** [comment]
Looks good, thank you!

File: src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs (hunk 1 above)
Comment (KonH) [reply src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs]: Thanks! I'm not sure about this change, push additional commit.

**Comment by KonH** [comment]
Now conflicts resolved (last commit).

**Pull request merged**